sections/home/WhatsHappening.jsx

```diff
@@ -2,12 +2,13 @@ import styled from "styled-components";
 
 import PumpDigest from "../../public/home/pump-digest.png";
 import ImpactReport from "../../public/home/impact-report.png";
-import { CardCarousel, Text } from "../../components";
+import { CardCarousel, Text, Loading } from "../../components";
 import { baseTheme } from "../../theme";
 import { media } from "../../utils";
 
 export const WhatsHappening = ({ ...props }) => (
   <Wrapper {...props}>
+    <Loading color = {baseTheme.colors.navy}></Loading>
```

Comment:
Since there are no children you should just be using ```<Loading color={...} />``` and it should be a self-closing tag.